spring-cloud-dataflow-docs/src/main/asciidoc/configuration-kubernetes-app-properties.adoc

```diff
@@ -1134,6 +1134,11 @@ The following example shows how you can configure an Init Container for an appli
 [source,options=nowrap]
```

Comment:
Should line 1131 be updated to say "The following example shows how you can configure an Init Container `or containers` for an application:"?

Reply:
Done